src/components/Properties/PropertyGroups.vue

```diff
@@ -109,6 +113,9 @@ export default {
 	},
 
 	computed: {
+		showAsText() {
+			return this.isReadOnly && this.localValue.length <= 1
```

Comment:
```suggestion
			return this.isReadOnly && this.localValue.length === 0
```

nitpick: easier to comprehend this way IMO

Reply:
It supposed to be true for single-values too

Reply:
Sorry, you are right. I misread as <1 but it's <=1